### PR TITLE
doc/performance

### PR DESCRIPTION
**Pull request by nucliweb**
- **docs(packages/sui-performance): Add sample and documentation to the Performance component**
- **chore(Root): update package-lock**

## Description

Add demo and screenshot samples to the Performance package

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -24856,37 +26892,6 @@
         "react-dom": "*"
       }
     },
-    "packages/sui-react-head/node_modules/@types/aria-query": {
```
Comment (andresz1) [inline package-lock.json:24859]: Is this needed ?

Comment (nucliweb) [reply package-lock.json:24859]: Let me check it running the phoenix script again

Comment (nucliweb) [reply package-lock.json:24859]: Rerunning the phoenix script only changes this...

image

**Review by andresz1 — APPROVED** [review]

**Comment by andresz1** [comment]
Nice 👏🏻

**Review by kikoruiz — APPROVED** [review]

**Pull request merged**